samples/dnn/optical_flow.py

```diff
@@ -46,7 +62,7 @@ def motion_to_color(self, flow):
         rad = rad[..., np.newaxis] / maxrad
         a = np.arctan2(-fy / maxrad, -fx / maxrad) / np.pi
         fk = (a + 1) / 2.0 * (ncols - 1)
-        k0 = fk.astype(np.int)
+        k0 = fk.astype(int)
```

Comment:
Keep np.int

Reply:
I suggest to use `np.int32` with specific bit suffix. `np.int` has been deprecated.

Reply:
I removed np.int because it was deprecated and found that `int` can be used in place for that. I looked it up and now I think np.int32 would be a better choice